[BUG]: typo in bump_version.yml

### What happened?
Git action failed with bump_version

### How can we reproduce issue?
Trigger bump version workflow

### Relevant log output
Run chmod +x ./version/update_version.sh
cat: ./build/version.txt: No such file or directory
Error: Process completed with exit code 1.

### Environment
Git

--- a/version/update_version.sh
+++ b/version/update_version.sh
@@ -2,7 +2,7 @@
 
 set -e
 
-current_version=$(cat ./build/version.txt)
+current_version=$(cat ./version/version.txt)
 
 new_version=""
 
@@ -22,4 +22,4 @@ case $1 in
     ;;
 esac
 
-echo $new_version > ./build/version.txt
+echo $new_version > ./version/version.txt

